qemu/tests/multi_disk_wild_hotplug.py

```diff
@@ -101,9 +112,11 @@ def _hotunplug_images():
         _hotplug_images()
         time.sleep(plug_time)
 
+    error_context.context("Wait some time %d" % wait_time, test.log.info)
```

Comment:
Suggest adding time units, i.e. second here. 
e.g. Waiting for some time: 30 seconds.

Reply:
updated, thanks